src/chart/bar/BarView.js

```diff
@@ -424,8 +424,8 @@ function updateStyle(
 
     el.useStyle(zrUtil.defaults(
         {
-            fill: color,
-            opacity: opacity
+            fill: layout.startAngle === layout.endAngle ? 'transparent' : color,
+            opacity: layout.startAngle === layout.endAngle ? 0 : opacity
```

Comment:
I think there is no need to set `opacity` to 0 if fill and stroke are both set to `'none'`

Reply:
`fill` : `transparent` , `opacity`: `0` , tooltip can be show if set,  i just feel it better. 
![image](https://user-images.githubusercontent.com/5095651/68186199-8292eb80-ffde-11e9-9fb3-a5ef611955e7.png)
Is that right？ @Ovilia

Reply:
I'm not sure if it's a good experience if cursor becomes to `poiner` and tooltips shows on an blank area. Perhaps users will think is it a bug that there should be something under the mouse but instead it's blank ?

Reply:
ACK, i commit again

Reply:
And `silent` property of the `el` should be set to `true` in case it displays again with emphasis style when user is hovering on the area.